homeassistant/core.py

```diff
@@ -1837,6 +1839,13 @@ def __init__(self, hass: HomeAssistant) -> None:
         # Use legacy template behavior
         self.legacy_templates: bool = False
 
+    @property
+    def language(self) -> str | None:
+        """Return the configured language."""
+        if self._language is not None:
+            return self._language
+        return "en"
```

Comment:
Why not making a default like `currency` ?

Reply:
That makes sense, changed accordingly 👍